.github/workflows/test.yml

```diff
@@ -1,6 +1,7 @@
 name: Test
 
 on:
+  workflow_dispatch:
```

Comment:
why is this needed? 🤔

Reply:
Good question, while testing, it was handy for me to just trigger a build without a PR or a push, I'll remove it!